gcloud/bigtable/cluster.py

```diff
@@ -317,11 +206,12 @@ def create(self):
         """
         request_pb = _prepare_create_request(self)
         # We expect a `google.longrunning.operations_pb2.Operation`.
-        operation_pb = self._instance._client._instance_stub.CreateCluster(
-            request_pb)
+        client = self._instance._client
+        operation_pb = client._instance_stub.CreateCluster(request_pb)
 
-        op_id = _process_operation(operation_pb)
-        return Operation('create', op_id, cluster=self)
+        operation = Operation.from_pb(operation_pb, client)
+        operation.target = self
+        return operation
```

Comment:
Oops, sorry:  I should have written "the type of the metadata instance".  Of course, two of our four APIs returning operations have *no* metadata instance at all. :(